srcpkgs/wildmidi/template

```diff
@@ -27,7 +26,7 @@ libwildmidi_package() {
 	conf_files="/etc/wildmidi/wildmidi.cfg"
 	pkg_install() {
 		vmove "usr/lib/*.so.*"
-		vinstall cfg/wildmidi.cfg 644 etc/wildmidi
+		vinstall ${FILESDIR}/wildmidi.cfg 644 etc/wildmidi
```

Comment:
`vconf` please, but does it really make sense to install the same conf file to /usr/share and /etc ? (vsconf in L20)

Reply:
I assume it was originally done that way so you can edit the /etc/wildmidi/wildmidi.cfg all you want while still having a backup available.

It is setup to have the config file in /etc/wildmidi/wildmidi.cfg by default so that apps that use it can work out of the box without having to create that config file manually.

I believe easyrpg needs /etc/wildmidi/wildmidi.cfg, but I could be mistaken. I can remove the L20 vsconf if you want.

Just asking @pullmoll for any comment

Reply:
How would you use vconf to install a configuration file to `/etc/wildmidi/`?